board-viewer.html

```diff
@@ -1739,7 +1740,19 @@
     this.fire('item-hover', info);
   },
 
+  _isTouchDevice: function() {
```

Comment:
This Function detect if we are in a touchscreen device... The only flat point i detected on this ,is that does not detect emulation switching in the browser tab, it must to be reloaded or loaded since the beginning in the device or emulation.